packages/block-library/src/query/utils.js

```diff
@@ -249,25 +244,29 @@ export function useBlockNameForPatterns( clientId, attributes ) {
 	const activeVariationName = useSelect(
 		( select ) =>
 			select( blocksStore ).getActiveBlockVariation(
-				queryLoopName,
+				'core/query',
 				attributes
 			)?.name,
 		[ attributes ]
 	);
-	const blockName = `${ queryLoopName }/${ activeVariationName }`;
-	const activeVariationPatterns = useSelect(
+	const blockName = `core/query/${ activeVariationName }`;
+	const hasActiveVariationPatterns = useSelect(
```

Comment:
Here again the code is interested only in the boolean value (is the list non-empty) not in the actual list.